packages/datastore/datastore/src/ModelSchema.ts

```diff
@@ -56,52 +103,100 @@ export class ModelSchema<T = any>{
     this.encrypted = extractEncryptedFields(this.fields, schema.encrypted);
   }
 
+  /**
+   * Method to fill the object with default values
+   *
+   */
   public fill(): void {
```

Comment:
It is redundant to have this method then? :)

Reply:
@wtrocki thinking about it further this method is redundant. We have a default value property though, which is not being used anywhere though.

Reply:
It is critical for us to implement migrations (this will be required for CLI etc.) Keep it :D